.github/actions/changes/action.yml

```diff
@@ -53,15 +53,15 @@ runs:
         github_token: ${{ inputs.token }}
     - uses: actions/checkout@v4
       with:
-        repository: ${{ steps.pr.outputs.source_repo }}
-        ref: ${{ steps.pr.outputs.source_branch }}
+        repository: ${{ github.repository }}
+        ref: ${{ steps.pr.outputs.sha }}
```

Comment:
We should checkout the sha on our repo here. Github also makes a magical `pull/ID/head` branch that will have the same sha as the source repo/branch HEAD. This should allow us to be in the correct repo when performing this check.